code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -818,6 +818,11 @@ obj/item/clothing/suit/cassock
 	body_parts_covered = ARMS|FULL_TORSO|IGNORE_INV
 	species_fit = list(VOX_SHAPED, INSECT_SHAPED)
 
+/obj/item/clothing/suit/mino/vinesafe
+
+/obj/item/clothing/suit/mino/vinseafe/vine_protected()
```

Comment:
if you typo the type path it won't work at all

also you don't need to have the blank definition above, it gets implied by the overridden proc

Reply:
both fixed